webpack/sass/main.scss

```diff
@@ -1,4 +1,18 @@
+
```

Comment:
Please remove the first empty line and only leave 1 empty line at the end and not two. And make these changes in every file please.

Reply:
Working on it.